src/utils.coffee

```diff
@@ -40,15 +40,13 @@ exports.mixpanelTrack = (event, properties = {}) ->
 
 	properties = _.cloneDeep(properties)
 
-	# Don't log private env vars (e.g. api keys)
-	if properties?.app?.env?
-		try
-			{ env } = properties.app
-			env = JSON.parse(env) if _.isString(env)
-			safeEnv = _.omit(env, config.privateAppEnvVars)
-			properties.app.env = JSON.stringify(safeEnv)
-		catch
-			properties.app.env = 'Fully hidden due to error in selective hiding'
+	# Don't log env vars as they may include keys or other secrets
+	if properties?.app?
+		delete properties.app.env
```

Comment:
I'd be safer to have a whitelist otherwise if we change this object the data will leak.

Reply:
hmm good point, good thing I hadn't merged this yet.
I'll change it to a whitelist, we'd just need to decide what elements of `app` and `stateDiff` are relevant for mixpanel.

Reply:
I propose picking these from app: `appId, imageId, commit, name`
And these from stateDiff (when the supervisor  fails to report current state): `status, download_progress, commit, os_version, supervisor_version, ip_address`

Make sense @petrosagg @pimterry @afitzek @izavits ? anything else you'd like to see or see removed?
This filters logs_channel (which is a sensitive field) and some fields that I think are irrelevant, like containerId, provisioning_progress, etc (I don't see how we'd use them on mixpanel).

Keep in mind much of these events will change soon with multicontainer (e.g. most of the things we currently associate with an "app" will now be associated with a "service", so we'll have "Service start" instead of "Application start", etc).

Reply:
I don't think that `provisioning_progress` will be needed for analytics, nor containerId. I think those fields you mention are fine.